packages/webdriverio/src/commands/browser/getWindowName.ts

```diff
@@ -0,0 +1,32 @@
+import getWindowNameHelper from '../../scripts/getWindowName'
+
+/**
+ *
+ * Get current active window name. This command is the equivalent function to `window.name`. This command does not
+ * work in mobile environments.
+ *
+ * __Note:__ When calling this command you gets current window name used during newWindow command.
+ *
+ * <example>
+    :getWindowName.js
+    it('should print window name', async () => {
+        await browser.url('https://google.com')
+        console.log(await browser.getTitle()) // outputs: "Google"
+
+        await browser.newWindow('https://webdriver.io', {
+            windowName: 'WebdriverIO window',
+            windowFeature: 'width=420,height=230,resizable,scrollbars=yes,status=1',
+        })
+        console.log(await browser.getWindowName()) // outputs: "WebdriverIO window"
+    });
+ * </example>
+ *
+ * @return {String}          name of active window
+ *
+ * @uses browser/execute
+ * @alias browser.getWindowName
+ * @type window
+ */
+export default async function getWindowName(this: WebdriverIO.Browser) {
```

Comment:
I don't think such a command would be helpful for WebdriverIO users. Given the amount of commands WebdriverIO provides we should be very cautions adding new ones. It seems that users, that are interested in the window name can just call the execute command themselves.